build/debian-hyperkube-base/Dockerfile

```diff
@@ -38,3 +34,8 @@ RUN echo CACHEBUST>/dev/null && clean-install \
     util-linux
 
 COPY cni-bin/bin /opt/cni/bin
+
+# The samba-common, cifs-utils, and nfs-common packages depend on
```

Comment:
I think you need to do this before the `clean-install` step, as otherwise the package configuration step fails.

Reply:
Looks like i have to install bash separately too before installing the rest. will rework